sos/report/plugins/ceph_common.py

```diff
@@ -41,57 +42,38 @@ class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
 
     # This check will enable the plugin regardless of being
     # containerized or not
-    files = (
-        '/etc/ceph/ceph.conf',
-    )
+    files = ('/etc/ceph/ceph.conf',)
 
     def setup(self):
         all_logs = self.get_option("all_logs")
 
         self.add_file_tags({
             '.*/ceph.conf': 'ceph_conf',
-            '/var/log/ceph/ceph.log.*': 'ceph_log',
-            '/var/log/ceph/ceph-osd.*.log': 'ceph_osd_log'
+            '/var/log/ceph/ceph.log*': 'ceph_log',
```

Comment:
by changing .* to * you are changing this from a regex to a glob. Is that intentional? The method at [1] suggests this should be a regex pattern.

[1] https://github.com/sosreport/sos/blob/aaeb8cb57ed55598ab744b96d4f127aedebcb292/sos/report/plugins/__init__.py#L1350

Reply:
thanks for spotting it, its a mistake. Shall correct it!